rotkehlchen/exchanges/kucoin.py

```diff
@@ -355,7 +354,7 @@ def _api_query_paginated(
             time_step = WEEK_IN_SECONDS
```

Comment:
It does not relate to you changes but as I see [here](https://github.com/rotki/rotki/pull/4410/files#diff-b88d93be9e15fa70b43c881ae449f53b5eaa251dfcbf5b5637e6aa47a12645c8R348) if `end_ts` is lower than `API_V2_TIMESTART`, data will be queried wrongly and will also include range (`end_ts`, `API_V2_TIMESTART`]. Am I right?